client/ionic/package.json

```diff
@@ -17,6 +17,7 @@
     "@types/react-dom": "^16.9.4",
     "@types/react-router": "^5.1.4",
     "@types/react-router-dom": "^5.1.3",
+    "dompurify": "^2.0.8",
```

Comment:
File an issue to review security settings on dompurify.  https://snyk.io/vuln/npm:dompurify